src/components/radio/RadioBox.tsx

```diff
@@ -65,16 +67,14 @@ export function RadioBox({
     >
       <ThemeText
         type="heading__title"
-        color={themeColor}
-        style={{...styles.title, marginBottom: spacing}}
+        style={[{...styles.title, marginBottom: spacing}, styleText]}
```

Comment:
Note that it is slightly preferable to send in a `ContrastColor` in the `color` param, as using styles is mainly used for overrides that is "outside" of the theme colors.

In this situation it would be something like:
```tsx
const themedColor = theme.interactive[interactiveColor][selected ? 'active' : 'default'];`
<ThemeText color={themedColor} />
```